packages/lesswrong/components/messaging/FriendlyInbox.tsx

```diff
@@ -0,0 +1,275 @@
+  rightColumn: {
+    flex: "1 1 auto",
+  },
+  hideColumnSm: {
+    [theme.breakpoints.down('xs')]: {
+      display: "none",
+    },
+  },
+  navigation: {
+    overflowY: "auto",
+    backgroundColor: theme.palette.background.pageActiveAreaBackground,
+    borderLeft: theme.palette.border.grey200,
+    borderRight: theme.palette.border.grey200,
+    height: "100%",
+  },
+  conversation: {
+    overflowY: "auto",
+    backgroundColor: theme.palette.background.pageActiveAreaBackground,
+    borderRight: theme.palette.border.grey200,
+    padding: "0px 32px",
+    flex: "1 1 auto",
+    [theme.breakpoints.down('xs')]: {
+      padding: "0px 24px",
+    },
+  },
+  columnHeader: {
+    backgroundColor: theme.palette.background.pageActiveAreaBackground,
+    border: theme.palette.border.grey200,
+    display: "flex",
+    alignItems: "center",
+    justifyContent: "space-between",
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    color: theme.palette.grey[1000],
+    fontSize: 20,
+    fontWeight: 600,
+    padding: 16,
+  },
+  columnHeaderLeft: {
+    borderTopLeftRadius: theme.borderRadius.default,
+    [theme.breakpoints.down('xs')]: {
+      borderTopLeftRadius: 0,
+    },
+  },
+  columnHeaderRight: {
+    borderLeft: "none",
+    borderTopRightRadius: theme.borderRadius.default,
+    [theme.breakpoints.down('xs')]: {
+      borderTopRightRadius: 0,
+      borderrRight: "none",
+    },
+  },
+  headerText: {
+    overflow: "hidden",
+    display: "-webkit-box",
+    "-webkit-box-orient": "vertical",
+    "-webkit-line-clamp": 1,
+    [theme.breakpoints.down('xs')]: {
+      "-webkit-line-clamp": 2,
+    }
+  },
+  actionIcon: {
+    color: theme.palette.grey[600],
+    width: 24,
+    height: 24,
+    cursor: "pointer",
+  }
+});
+
+const FriendlyInbox = ({
+  currentUser,
+  terms,
+  conversationId,
+  isModInbox = false,
+  classes,
+}: InboxComponentProps & {
+  conversationId?: string;
+}) => {
+  const { openDialog } = useDialog();
+  const { currentRoute, location } = useLocation();
+  const { history } = useNavigation();
+
+  const selectedConversationRef = useRef<HTMLDivElement>(null);
+
+  const selectConversationCallback = useCallback(
+    (conversationId: string | undefined) => {
+      history.push({ ...location, pathname: `/${isModInbox ? "moderatorInbox" : "inbox"}/${conversationId}` });
+    },
+    [history, isModInbox, location]
+  );
+
+  const openNewConversationDialog = useCallback(() => {
+    openDialog({
+      componentName: "NewConversationDialog",
+      componentProps: {
+        isModInbox,
+      },
+    });
+  }, [isModInbox, openDialog]);
+
+  const { InboxNavigation2, ConversationWidget, ForumIcon, ConversationDetails, Typography } = Components;
+
+  const conversationsResult: UseMultiResult<"ConversationsList"> = useMulti({
+    terms,
+    collectionName: "Conversations",
+    fragmentName: "ConversationsList",
+    limit: 500,
+  });
+  const { results: conversations } = conversationsResult;
```

Comment:
There is a bug here where just after you create a conversation it doesn't appear in this list because:
 - this query doesn't refetch
 - there are no messages in the conversation

I've made a ticket to follow up on this: https://app.asana.com/0/628521446211730/1205817215822664/f